### ISSUE TITLE
Doesn't refresh state.

### ISSUE THREAD

**Issue by velkhor** [Lyr3x/hass-Duet3D#19]
I install this plugin, connect to printer (SBC mode - checkbox unchecked of course).
Any data from printer does'nt update. I still see data from first connect.

**Comment by Lyr3x** [comment]
Does it never update? Don't have a duet printer at hand currently. Any error logs ?

**Comment by velkhor** [comment]
I have Duet (in build but almost finished). Board BTT Octopus Pro F429, raspbian installed on Pi SBC mode.
When I install plugin and reboot my printer, stats updated, but only once. I don't have extruder yet, but heat bed is installed. I try heating and cooling bed, but i don't see updates of temp.
I try print also (only for cariage moves) but no update print progress, don't change X,Y,Z coordinates.

**Comment by CrazyCreator13** [comment]
Don't update if you reload the page?

**Comment by velkhor** [comment]
Yes don't update. Update only when i reboot HA. In other case don't update.
Here is bed temp chart. I restart HA where you see changes to 0 and 50.
![Zrzut ekranu z 2024-01-04 19-37-13](https://github.com/Lyr3x/hass-Duet3D/assets/8623886/633a6e8f-9d0e-4030-9363-9486bc0fb99d)
Printer was on to 02:00 at night...

**Comment by CrazyCreator13** [comment]
![Bildschirmfoto 2024-01-05 um 19 48 34](https://github.com/Lyr3x/hass-Duet3D/assets/3821732/b578941d-2f45-46f7-9e82-2fde2a42ff83)
It doesn't update every second, but I would say every 30 seconds

**Comment by velkhor** [comment]
Yes. I know not every second... but i have 4 updates in 5 hours.
Every update executed on reboot HA.

**Comment by Diamondback2010** [comment]
Same here, but in **standalone** mode. It only ever updates once when rebooting or re-configuring. I tried variousupdate intervals to no avail.

**Comment by Lyr3x** [comment]
I have a working printer with Duet board again, and can now debug this now

**Comment by Lyr3x** [comment]
I have a working printer with RRF and a Duet3 in SBC again, so I can take a closer look. I currently cannot reproduce the issue though. 
![image](https://github.com/Lyr3x/hass-Duet3D/assets/6078365/ce6c320e-563f-41e4-980c-9ec62e87ccdd)
![image](https://github.com/Lyr3x/hass-Duet3D/assets/6078365/3d7977b7-7e27-430d-bac3-a29fa84de76f)
![image](https://github.com/Lyr3x/hass-Duet3D/assets/6078365/916b2265-452a-4ac0-8d1a-6b229bf35cf9)
![image](https://github.com/Lyr3x/hass-Duet3D/assets/6078365/753a61a9-08f1-40da-b97f-2f96789064d3)


Can you give me more details and check your logs?
